code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -24,4 +24,3 @@
 	new /obj/item/device/assembly/flash/handheld(src)
 	new /obj/item/device/laser_pointer(src)
 	new /obj/item/door_remote/research_director(src)
-	new /obj/item/device/firing_pin/test_range(src)

```

Comment:
Why did you change them to test range firing pins and then remove the test range firing pins

Reply:
Because Kor hates test firing pins